util/argo/audit_logger.go

```diff
@@ -94,7 +94,7 @@ func (l *AuditLogger) logEvent(objMeta ObjectRef, gvk schema.GroupVersionKind, i
 	}
 }
 
-func (l *AuditLogger) LogAppEvent(app *v1alpha1.Application, info EventInfo, message, user string) {
+func (l *AuditLogger) LogAppEvent(app *v1alpha1.Application, info EventInfo, spec v1alpha1.ApplicationSpec, message, user string) {
```

Comment:
We can actually avoid the extra parameter by just using `app.Spec` instead of the new `spec`. Can't believe I didn't notice this before.  :-P